griptape/drivers/vector/opensearch_vector_store_driver.py

```diff
@@ -93,7 +93,7 @@ def load_entry(self, vector_id: str, *, namespace: Optional[str] = None) -> Opti
             else:
                 return None
         except Exception as e:
-            logging.error(f"Error while loading entry: {e}")
+            logging.exception("Error while loading entry: %s", e)
```

Comment:
I was going to complain about not using f-strings, but the argument in [G004](https://docs.astral.sh/ruff/rules/logging-f-string/) about deferring string formatting speaks to me. TIL

(Would be even cooler if the formatted values could be lazily computed too)